src/transactions/OperationFrame.cpp

```diff
@@ -153,6 +160,15 @@ OperationFrame::checkValid(SignatureChecker& signatureChecker, Application& app,
                            LedgerDelta* delta)
 {
     bool forApply = (delta != nullptr);
+    if (!isVersionSupported(app.getLedgerManager().getCurrentLedgerVersion()))
```

Comment:
it is bad that we cannot move MANAGE_DATA_NOT_SUPPORTED_YET here as well

Reply:
yeah `MANAGE_DATA_NOT_SUPPORTED_YET` should not have been implemented like this in the first place and now we have to keep that behavior to replay old versions of the protocol